dg/commands/cpd4/install.py

```diff
@@ -58,7 +61,15 @@
     ),
 )
 @click.option("--project", default="zen", help="Project where the Cloud Pak for Data control plane is installed")
-@click.option("--storage-class", help="Storage class used for installation", required=True)
+@click.option("--storage-class", help="Storage class used for installation")
+@click.option(
+    "--storage-vendor",
+    help="Storage type used for installation",
+    type=click.Choice(
+        list(map(lambda x: x.name.lower(), CloudPakForDataStorageVendor)),
+        case_sensitive=False,
+    ),
+)
```

Comment:
Do we need to add some property like notRequiredIf for the  `--storage-class` and `--storage-class`. There is a good example of that in [this thread](https://stackoverflow.com/a/44349292) in Stackoverflow. This might be useful for further commands as well.